astroid/brain/brain_builtin_inference.py

```diff
@@ -641,12 +646,13 @@ def infer_property(
 
     prop_func = objects.Property(
         function=inferred,
-        name=inferred.name,
+        name="<property>",
         lineno=node.lineno,
         col_offset=node.col_offset,
+        # ↓ semantically, the definition of this property isn't within
+        # node.frame (or anywhere else, really)
+        parent=AstroidManager().adhoc_module,
```

Comment:
I don't understand this change? Why doesn't the property have the parent of the function being decorated as its parent?

Reply:
See the comment above, this is about «inplace» properties, not the «decorator» properties.